README.md

```diff
@@ -310,7 +311,7 @@ This section describes the flags common across all the subcommands. For flags
 specific to a give subcommand run `harbourbridge help <subcommand>`.
 
 `-source` Required flag. Specifies the source database. Supported source
-databases are _'postgres'_, _'mysql'_ and _'dynamodb'_.
+databases are _'postgres'_, _'mysql'_, _'dynamodb'_ and _'csv'_(only in data mode).
```

Comment:
change this to Specifies the data source as CSV is not a database.

Reply:
Good catch